2020/day-14/AngelOnFira/day14.py

```diff
@@ -0,0 +1,45 @@
+import re
+import itertools
+
+data = [line.strip() for line in open("input.txt", 'r')]
+
+count = 0
+pos = [0, 0]
+
+memory = {}
+mask = ""
+
+for i, value in enumerate(data):
+    if value.startswith("mask ="):
+        mask_str = value.split(" = ")
+        mask = mask_str[1]
+
+    else:
+        groups = re.search(r'mem\[(\d+)\] = (\d+)', value)
+        location = int(groups.group(1))
+        value = int(groups.group(2))
+
+        this_loc = ""
+
+        binary = '{0:036b}'.format(location)
```

Comment:
I lost it here in my solution since I didn't pad the 36 properly, found it quickly tho so no harm done

Reply:
This is something that would have gotten me running in loops for a while I think. Luckily the first stack overflow I found had this template :P